dataset/data.go

```diff
@@ -82,10 +82,11 @@ type Event struct {
 
 // CodeList holds one of the codelists corresponding to a new Instance
 type CodeList struct {
-	ID          string `json:"id"`
-	HRef        string `json:"href"`
-	Name        string `json:"name"`
-	IsHierarchy bool   `json:"is_hierarchy"`
+	ID                    string `json:"id"`
+	HRef                  string `json:"href"`
+	Name                  string `json:"name"`
+	IsHierarchy           bool   `json:"is_hierarchy"`
+	IsCantabularGeography bool   `json:"is_cantabular_geography"`
```

Comment:
I don't actually think this field (or isHierarchy) are valid in the dataset model for CodeList. 

Dataset API responses only link to the CodeList. This struct seems to be used in place of Dimension, in the []Dimension returned on a /version or /instance endpoint. e.g. https://api.beta.ons.gov.uk/v1/datasets/wellbeing-quarterly/editions/time-series/versions/3 

those responses look like this:
```
{
"href": "https://api.beta.ons.gov.uk/v1/code-lists/yyyy-qq",
"id": "yyyy-qq",
"label": "Time",
"links": {
"code_list": {},
"options": {},
"version": {}
},
"name": "time"
},
```

and don't currently contain. the isHierarchy field. If we need to be able to store this data in the dataset API, then I'd suggest removing this line from this PR and making the change to the Dataset API to allow the field first - then come back and add this when it's a recognized field :)